io-ballerina/src/io/character/channel_string_io.bal

```diff
@@ -208,14 +208,14 @@ public function channelWriteJson(WritableChannel writableChannel, json content)
     }
 }
 
-# Write XML content to a channel.
+# Write XML content to a channel
```

Comment:
Again, how is the XML serialized? The encoding used?

Reply:
Here all APIs are using default encoding as `UTF-8`. We are not concerned about other encoding types here.